samples/globalapp/Dockerfile.aot-sdk-publish.alpine

```
@@ -0,0 +1,8 @@
+# This image allows easily building and publishing AOT-compiled .NET applications
+# To use:
+# * mount the Docker socket: `-v /var/run/docker.sock:/var/run/docker.sock`
+# * mount the source code directory: `-v /path/to/source:/source`
+# * call `dotnet publish -t PublishContainer` with/on a project that 
+#   has been configured for AOT compilation 
+FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/sdk:10.0-alpine-aot AS builder
+RUN apk add --no-cache docker
```

Since you're using the host's Docker socket you only need the `docker-cli` package:

```suggestion
RUN apk add --no-cache docker-cli
```